packages/clerk-js/src/ui.retheme/elements/Select.tsx

```diff
@@ -134,18 +140,19 @@ export const Select = withFloatingTree(<O extends Option>(props: PropsWithChildr
   );
 }) as <O extends Option>(props: PropsWithChildren<SelectProps<O>>) => ReactElement;
 
-type SelectOptionBuilderProps<O extends Option> = {
+type SelectrenderOptionProps<O extends Option> = {
   option: Option;
   index: number;
-  optionBuilder: OptionBuilder<O>;
+  renderOption: RenderOption<O>;
   handleSelect: (option: Option) => void;
-  isFocused: boolean;
+  isFocused?: boolean;
+  isSelected?: boolean;
```

Comment:
These will always be passed from our component, why did we decided to make them optional ?

Reply:
Component-wise, it makes more sense as they're not required.